diff --git a/pywake_api.py b/pywake_api.py
 
index 903d4a9..8cbbbe5 100644
 
--- a/pywake_api.py
 
+++ b/pywake_api.py
 
@@ -21,6 +21,7 @@ from py_wake import NOJ, BastankhahGaussian
 
from py_wake.deficit_models import SelfSimilarityDeficit2020
 
from py_wake.superposition_models import LinearSum, SquaredSum
 
from py_wake.rotor_avg_models import RotorCenter, GridRotorAvg, EqGridRotorAvg, GQGridRotorAvg, CGIRotorAvg, PolarGridRotorAvg, polar_gauss_quadrature, GaussianOverlapAvgModel
 
+from py_wake.deficit_models.utils import ct2a_mom1d
 
 
 
 
 
# Define default values for wake_model parameters
 
@@ -255,6 +256,7 @@ def run_pywake(yamlFile, output_dir='output'):
 
        deficit_param_mapping = {'k': 'k'}
 
     elif wake_model_data['name'] == 'Bastankhah':
 
        wakeModel = BastankhahGaussianDeficit
 
+       deficit_args['ct2a'] = ct2a_mom1d
 
        deficit_param_mapping = {'k': 'k', 'ceps': 'ceps'}
 
     elif wake_model_data['name'].upper() == 'FUGA':
 
        wakeModel = FugaDeficit
